### PR TITLE
chore: add cypress tests

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const renameKeys = o =>
+  Object.keys(o || {}).reduce(
+    (acc, key) =>
+      Object.assign({}, acc, {[key.replace(/^CYPRESS_/, '')]: o[key]})
```
Comment (khanlifts) [inline cypress/plugins/index.js]: Doesn't this work with the spread operator too?

File: cypress/plugins/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const renameKeys = o =>
```
Comment (khanlifts) [inline cypress/plugins/index.js]: ES5 or ES6 in cypress directory?

File: cypress/plugins/index.js (hunk 1 above)
Comment (MethodenMann) [reply cypress/plugins/index.js]: Nope. not supported in this context

File: cypress/plugins/index.js (hunk 2 above)
Comment (MethodenMann) [reply cypress/plugins/index.js]: ES6

**Pull request merged**